src/main/kotlin/com/dylanmiska/RoboPantryAPI/adapter/persistence/entity/purchase/PurchaseEntity.kt

```diff
@@ -12,14 +10,11 @@ import java.util.*
 @Where(clause = "expired_id IS NULL")
```

Comment:
This will eventually need to change. Expired is really a partial value for a purchase since some items can be used up in the pantry while others not.